src/sweep_event.js

```diff
@@ -113,16 +117,25 @@ export default class SweepEvent {
   }
 
 
+  /**
+   * Does event belong to result?
+   * @return {Boolean}
+   */
+  inResult() {
+    return this.resultTransition !== 0;
+  }
```

Comment:
Could possibly use a [getter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get)
````
getter inResult() {
    return this.resultTransition !== 0;
  }
````
This would allow you to call `something.inResult` like a property rather than having to swap to a method.

This is probably a minor quibble

Reply:
I think it doesn't have to be calculated dynamically and can be turned into a flag.

Reply:
Good idea with the getter => adapted.

@w8r Note that `inResult` is only needed in one place now (at the top of the `orderEvents` function), so the number of `inResult` calls isn't significant. Maybe the additional memory of storing the value might even slow things down.